Pod/Classes/NWSTokenView.swift

```diff
@@ -47,8 +40,33 @@ public class NWSTokenView: UIView, UIScrollViewDelegate, UITextViewDelegate
     
     // MARK: Private Vars
     private var shouldBecomeFirstResponder: Bool = false
-    private var scrollView = UIScrollView()
+    public var scrollView = UIScrollView()
     public var textView = UITextView()
+    public var toTextColor = UIColor.blackColor() {
+        didSet {
+            self.label.textColor = toTextColor
+        }
+    }
+    public var mainTextColor = UIColor.blackColor() {
+        didSet {
+            self.textView.textColor = mainTextColor
+        }
+    }
+    public var placeholderTextColor = UIColor.lightGrayColor() {
+        didSet {
+            self.textView.textColor = placeholderTextColor
+        }
+    }
+    public var defaultFontToLabel = UIFont(name: "HelveticaNeue", size: 14) {
```

Comment:
@wfilleman Please change to 'labelFont'. No need for 'default' since this property will be used regardless of whether user adjusts it.